settings.yaml.template

```diff
@@ -97,6 +98,9 @@ AWS:
             DELETE_STACK: 'test'
             # Number of minutes the deletable CloudFormation should be allowed to live, e.g 120 minutes = 2 Hours
             SLA_MINUTES: 120
+        OCPS:
+            # Specified as {time_value}{time_unit} format, e.g. "7d" = 7 Days
```

Comment:
Also mention that if the unit is not mentioned it would be considered as `minute`.